services/app-web/src/constants/routes.ts

```diff
@@ -29,23 +30,25 @@ const STATE_SUBMISSION_FORM_ROUTES = [
     'SUBMISSIONS_CONTACTS',
     'SUBMISSIONS_DOCUMENTS',
     'SUBMISSIONS_REVIEW_SUBMIT',
+    'SUBMISSIONS_SUMMARY',
 ] as RouteT[]
 
 const RoutesRecord: Record<RouteT, string> = {
     ROOT: '/',
     AUTH: '/auth',
     DASHBOARD: '/dashboard',
     HELP: '/help',
-    SUBMISSIONS: '/submissions',
-    SUBMISSIONS_NEW: '/submissions/new',
-    SUBMISSIONS_FORM: '/submissions/:id',
-    SUBMISSIONS_TYPE: '/submissions/:id/type',
-    SUBMISSIONS_CONTRACT_DETAILS: '/submissions/:id/contract-details',
-    SUBMISSIONS_RATE_DETAILS: '/submissions/:id/rate-details',
-    SUBMISSIONS_CONTACTS: '/submissions/:id/contacts',
-    SUBMISSIONS_DOCUMENTS: '/submissions/:id/documents',
-    SUBMISSIONS_REVIEW_SUBMIT: '/submissions/:id/review-and-submit',
-    SUBMISSIONS_REVISION: '/submissions/:id/revisions/:revisionVersion',
+    SUBMISSIONS: 'submissions',
+    SUBMISSIONS_NEW: 'submissions/new',
+    SUBMISSIONS_FORM: 'submissions/:id/form/*',
```

Comment:
For visibility - This PR nests all state submission form related fields under `submissions/:id/form`.  This is a significant change to our routes and will need design approval. 

@JasonLin0991 and I talked about this change a a bit - he pointed out that we might be nesting too much under `submissions/:id`. In addition, we may have many other routes nested under `submissions/"id` in the future, particularly relating the CMS review process.

Reply:
These new routes /form and /summary will definitely need buy in from design. I think /form/ (or a different name) could be a good change b/c it nests all the pages of the form in one place. I’m less convinced that /summary is a good move b/c it seems pretty clear that going to /submissions/:id will show us a summary of the form. We should be able to figure out how to get the routs to match whatever scheme design desicds they want here.

Reply:
for visibility - after discussion we decided to go with `submissions/:id/edit` instead of form.